bin/generate-pending-release-diffs.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# Generate an overview of the plugin changes between the local filesystem and the latest stable (trunk) versions
+# committed to SVN in the WordPress.org Plugin Directory. The output is in Markdown format which is suitable for
+# pasting into a GitHub release preparation PR. Program status updates are sent to STDERR, so STDOUT can be piped
+# either to the clipboard or to a file for posting to GitHub.
+#
+# USAGE:
+# ./generate-pending-release-diffs.sh > overview.md
+# ./generate-pending-release-diffs.sh | pbcopy
+# npm run generate-pending-release-diffs --silent
+
+set -e
+
+for required_command in npm git svn jq rsync; do
+	if ! command -v "$required_command" &> /dev/null; then
+		echo "Error: The $required_command command must be installed to use this script." >&2
+		exit 1
+	fi
+done
+
+cd "$(git rev-parse --show-toplevel)"
+
+npm run build:plugin:performance-lab >&2 # TODO: Remove this as of <https://github.com/WordPress/performance/pull/1182>.
+npm run build-plugins >&2
+
+stable_dir=/tmp/stable-svn
+mkdir -p "$stable_dir"
+for plugin_slug in $(jq '.plugins[]' -r plugins.json) 'performance-lab'; do
```

Comment:
Remove with merging #1182.

```suggestion
for plugin_slug in $(jq '.plugins[]' -r plugins.json); do
```